frontend/simple/components/GroupSupportHistory.vue

```diff
@@ -59,6 +61,9 @@ h3.title {
         if (percentage < 1) return '#F6A623'
         return '#9CD445'
```

Comment:
More orphaned mystery values.

Remember, we're going to have themes in the future, so we can't have these raw values hanging around here. They must come from the CSS. See also other comment above about same thing. @sandrina-p can advise on how to integrate CSS <-> JS

Reply:
ACK this isn't part of this PR and isn't your doing, so I opened a separate issue for it: #373

(sidenote: this can be rewritten a bit more cleanly as `return percentage < 1 ? 'color-1' : 'color-2'`)